pkg/operation/botanist/component/resourcemanager/resource_manager_test.go

```diff
@@ -2342,8 +2376,6 @@ subjects:
 			BeforeEach(func() {
 				cfg.TargetDiffersFromSourceCluster = false
 				cfg.WatchedNamespace = nil
-				configMap = configMapFor(nil, pointer.String(gardenerutils.PathGenericKubeconfig))
-				deployment = deploymentFor(configMap.Name, cfg.KubernetesVersion, nil, pointer.String(gardenerutils.PathGenericKubeconfig), false, nil)
```

Comment:
Why was this removed?

Reply:
See https://github.com/gardener/gardener/pull/7191/files#r1093458598